ebos/eclgenericoutputblackoilmodule.cc

```diff
@@ -1138,9 +1164,69 @@ doAllocBuffers(unsigned bufferSize,
     rstKeywords["PRES"] = 0;
     rstKeywords["PRESSURE"] = 0;
 
-    // If TEMP is set in RPTRST we output temperature even if THERMAL
-    // is not activated
-    if (enableEnergy_ || rstKeywords["TEMP"] > 0) {
```

Comment:
We need to be a little careful that we preserve this condition.  It was introduced in #4646, i.e., after the current PR was created, but it's mostly a matter of resolving a merge conflict I think.